CONTRIBUTING.md

```diff
@@ -95,7 +96,9 @@ You can choose from a list of sub-dependent repos to contribute to, a few highli
 - [Chaos-workflows](https://github.com/litmuschaos/chaos-workflows)
 - [Test-tools](https://github.com/litmuschaos/test-tools)
 - [Litmus-go](https://github.com/litmuschaos/litmus-go)
-- [website-litmuschaos](https://github.com/litmuschaos/website-litmuschaos)
+- [Litmus-website](https://github.com/litmuschaos/litmus-website-2)
+- [Litmusctl](https://github.com/litmuschaos/litmusctl)
+- [Litmus-docs](https://github.com/litmuschaos/litmus-docs)
 
```

Comment:
How about adding `- [backstage-plugin](https://github.com/litmuschaos/backstage-plugin)` here?

Reply:
@namkyu1999 I have added the changes, PTAL